pkg/controllers/nsxserviceaccount/nsxserviceaccount_controller.go

```diff
@@ -177,7 +190,28 @@ func (r *NSXServiceAccountReconciler) GarbageCollector(cancel chan bool, timeout
 		}
 		gcSuccessCount, gcErrorCount := r.garbageCollector(nsxServiceAccountUIDSet, nsxServiceAccountList)
 		log.V(1).Info("gc collects NSXServiceAccount CR", "success", gcSuccessCount, "error", gcErrorCount)
+		count, ca = r.validateRealized(count, ca, nsxServiceAccountList)
```

Comment:
It seems it needs to wait for GCInterval (60s) before the first GC action. I think 60s delay is OK for client cert rotation. On the other hand, when NSX ca cert is updated, is it better to propagate the new ca cert to all NSXSAs as soon as possible?

Reply:
Fixed code and updated PR description.